prod/westeurope/cgn/functions_cgn_merchant/function_app_slot_staging/terragrunt.hcl

```diff
@@ -0,0 +1,117 @@
+dependency "subnet" {
+  config_path = "../subnet"
+}
+
+# cgn
+dependency "resource_group" {
+  config_path = "../../resource_group"
+}
+
+dependency "cosmosdb_cgn_account" {
+  config_path = "../../cosmosdb/account"
+}
+
+dependency "cosmosdb_cgn_database" {
+  config_path = "../../cosmosdb/database"
+}
+
+dependency "storage_account_cgn" {
+  config_path = "../../storage_cgn/account"
+}
+
+dependency "redis" {
+  config_path = "../../redis/redis_cache"
+}
+
+# Common
+dependency "virtual_network" {
+  config_path = "../../../common/virtual_network"
+}
+
+dependency "application_insights" {
+  config_path = "../../../common/application_insights"
+}
+
+dependency "key_vault" {
+  config_path = "../../../common/key_vault"
+}
+
+dependency "subnet_azure_devops" {
+  config_path = "../../../common/subnet_azure_devops"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_function_app?ref=v2.1.35"
```

Comment:
this module is wrong .... for staging **azurerm_function_app_slot** is required.